builder/files/bor.service

```diff
@@ -6,11 +6,39 @@
 [Service]
   Restart=on-failure
   RestartSec=5s
-  WorkingDirectory=$NODE_DIR
-  EnvironmentFile=/etc/matic/metadata
-  ExecStart=/usr/local/bin/bor $VALIDATOR_ADDRESS
+  ExecStart=/usr/local/bin/bor \
```

Comment:
Can we make use of `--bor-mainnet` and `--bor-mumbai` instead?

Reply:
I don't know, do they work correctly?

Reply:
Yes, it works perfectly. We have collected all the common params in there. You can compare it with those configs. Also, if there are additional configs or want to override any values, you can specify them after `--bor-mainnet`.

Reply:
this is great, then I think we can set this to `--bor-mumbai` by default and leave a comment on `--bor-mainnet`

Reply:
According to https://github.com/maticnetwork/bor/blob/7368ca6888c6c4850dce537d4aebf9229ba1a55b/cmd/geth/config.go#L342

the deafult sync mode is `snap`, should be `full` right?

Reply:
Yes, it should be `full`.

Reply:
Changed in defaults